torch/distributed/tensor/parallel/style.py

```diff
@@ -131,11 +153,38 @@ def make_input_shard_1d_dim_last(
             Default: ``None``
 
     Returns:
-        A :class:`DTensor` sharded on dimension ``dim`` over ``device_mesh``.
+        A :class:`DTensor` sharded on dimension ``-1`` over ``device_mesh``.
     """
     return make_input_shard_1d(input, device_mesh, dim=-1)  # type: ignore[call-arg]
 
 
+def make_input_reshard_replicate(
+    input: torch.Tensor,
+    device_mesh: DeviceMesh,
+) -> DTensor:
+    """
+    To convert a Tensor on different ranks to a Sharded DTensor
+    and then convert to a replicate DTensor.
+
+    Args:
+        input (:class:`torch.Tensor`):
+            This single tensor consists of a global DTensor sharded on dimension ``0``
```

Comment:
why a single tensor would consists a global DTensor?

I guess it should be sth like: "The input tensor on each rank which could form a global DTensor ..."